CONTRIBUTING.md

```diff
@@ -1,28 +1,64 @@
-# How to Contribute
+====================
+Contributing to CDAP.io website
+====================
 
-We'd love to accept your patches and contributions to this project. There are
-just a few small guidelines you need to follow.
+Here are instructions to get you started if you want to contribute to CDAP.io website. 
 
-## Contributor License Agreement
+Build Environment
+=================
 
-Contributions to this project must be accompanied by a Contributor License
-Agreement. You (or your employer) retain the copyright to your contribution;
-this simply gives us permission to use and redistribute your contributions as
-part of the project. Head over to <https://cla.developers.google.com/> to see
-your current agreements on file or to sign a new one.
+For instructions on setting up your development environment, please see the
+`README.md <https://github.com/cdapio/website/blob/develop/README.md>`__ file.
 
-You generally only need to submit a CLA once, so if you've already submitted one
-(even if it was for a different project), you probably don't need to do it
-again.
+Contribution Guidelines
+=======================
 
-## Code reviews
+Creating Issues
+---------------
+An issue should be created at http://github.com/cdapio/website for any bugs or new features before anybody starts working on it. 
+Please take a moment to check that an issue doesn't already exist before creating a new issue. 
+If you see that a bug or an improvement exists already, add a "+1" to the issue to indicate that you have the same
+problem as well. This will help us in prioritization.
 
-All submissions, including submissions by project members, require review. We
-use GitHub pull requests for this purpose. Consult
-[GitHub Help](https://help.github.com/articles/about-pull-requests/) for more
-information on using pull requests.
+Pull Requests
+-------------
+We love having pull requests and we do our best to review them as quickly as possible. 
+If your pull request is not accepted on the first try, don't be discouraged. 
+If there is a problem with the implementation, you will receive feedback on what to improve.
+Add a link to your PR on the issue, and in an email to `cdap-dev@googlegroups.com <mailto:cdap-dev@googlegroups.com>`__.
 
-## Community Guidelines
+Conventions
+-----------
+* Fork the repository and make changes on your fork in a feature branch.
 
-This project follows
```

Comment:
I think it might be ok to leave the community guidelines section as a separate section.